drivers/wifi/nrf700x/src/shim.c

```diff
@@ -27,19 +28,73 @@
 #include "qspi_if.h"
 
 LOG_MODULE_REGISTER(wifi_nrf, CONFIG_WIFI_NRF700X_LOG_LEVEL);
+K_HEAP_DEFINE(wifi_drv_mgmt_mem_pool, CONFIG_NRF_WIFI_CONFIG_HEAP_MEM_POOL_SIZE);
```

Comment:
Should we use `K_HEAP_DEFINE_NOCACHE` instead? In case some system supports caching, I don't think our driver will work?

Reply:
Might also need to evaluate any alignment requirements, i.e., using `k_heap_aligned_alloc`?

Reply:
Done